utils/build-script

```diff
@@ -763,8 +763,8 @@ class BuildScriptInvocation(object):
         # add them as one command.
         if args.extra_swift_args:
             impl_args += [
-                "--extra-swift-args",
-                ";".join(args.extra_swift_args)]
+                "--extra-swift-args=%s" % ';'.join(args.extra_swift_args)
```

Comment:
We should probably use the more modern [`.format()`](https://docs.python.org/3.4/library/functions.html#format) in this case, but it's not entirely necessary as the rest of the file doesn't.